src/meta-srv/src/metasrv.rs

```diff
@@ -196,6 +198,11 @@ impl MetaStateHandler {
         if let Err(e) = self.procedure_manager.start().await {
             error!(e; "Failed to start procedure manager");
         }
+
+        if let Err(e) = self.wal_options_allocator.start().await {
+            error!(e; "Failed to start wal options allocator");
+        }
```

Comment:
Nit(Non-blocking): 
We should ensure the start will only be executed once in the life cycle. We can improve this in the future.